.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+
+  "rules": {
+    // Maybe enable?
+    "one-var": 0,
+    "vars-on-top": 0,
+    "eqeqeq": 0,
+    "yoda": [0, "never"],
```

Comment:
this is fine.  it can reduce bugs where assignments were supposed to be conditionals, but that is pretty rare